internal/jobqueue/jobqueuetest/jobqueuetest.go

```diff
@@ -814,5 +819,107 @@ func testFail(t *testing.T, q jobqueue.JobQueue) {
 		require.Less(t, startTime, tmr)
 		require.Greater(t, endTime, tmr)
 	}
+}
+
+// sortUUIDs is a helper to sort a list of UUIDs
+func sortUUIDs(entries []uuid.UUID) {
+	sort.Slice(entries, func(i, j int) bool {
+		return entries[i].String() < entries[j].String()
+	})
+}
```

Comment:
I would probably define the `func(i, j int)` as a helper and use it on multiple callsites but whatever, this works too :-D